packages/react/src/test.setup.ts

```diff
@@ -18,3 +18,5 @@ indexStructureDefinitionBundle(readJson('fhir/r4/profiles-resources.json') as Bu
 indexStructureDefinitionBundle(readJson('fhir/r4/profiles-medplum.json') as Bundle);
 indexSearchParameterBundle(readJson('fhir/r4/search-parameters.json') as Bundle<SearchParameter>);
 indexSearchParameterBundle(readJson('fhir/r4/search-parameters-medplum.json') as Bundle<SearchParameter>);
+
+Object.defineProperty(globalThis.window, 'sessionStorage', { value: new MemoryStorage() });
```

Comment:
Why was this necessary?

Reply:
`sessionStorage` does not exist in the `jsdom` environment